deploy/addons/helm-tiller/README.md

```diff
@@ -8,16 +8,16 @@ To enable this addon, simply run:
 minikube addons enable helm-tiller
 ```
 
-In a minute or so tiller will be installed into your cluster. You could run `helm init` each time you create a new minikube instance or you could just enable this addon.
-Each time you start a new minikube instance tiller will be automatically installed. 
+In a minute or so tiller will be installed into your cluster. You could run `helm init` each time when you create a new minikube instance or you could just enable this addon.
```

Comment:
This one is incorrect.

Reply:
Please tell me in detail.
Thank you.

Reply:
I'm not clear what detail you want; it just sounds wrong.

Compare the [relative occurrence in books](
https://books.google.com/ngrams/graph?content=each+time+you%2Ceach+time+when+you&year_start=1900&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Ceach%20time%20you%3B%2Cc0%3B.t1%3B%2Ceach%20time%20when%20you%3B%2Cc0#t1%3B%2Ceach%20time%20you%3B%2Cc0%3B.t1%3B%2Ceach%20time%20when%20you%3B%2Cc0) of "each time you" and "each time when you".

Reply:
Ok.